vhost_user_block/src/lib.rs

```diff
@@ -270,8 +270,8 @@ impl VhostUserBackend for VhostUserBlkBackend {
         debug!("event received: {:?}", device_event);
 
         match device_event {
-            q if device_event < self.config.num_queues => {
-                let mut vring = vrings[q as usize].write().unwrap();
+            q if q < self.config.num_queues => {
+                let mut vring = vrings[0].write().unwrap();
```

Comment:
Why are you checking this value and ignoring it?

Reply:
device_event is the event index registered on epoll_fd, it's equal to the original queue index. However, while create multiple worker threads in vhost_user lib, for vhost_user_blk, it's assumed that one thread will handle one queue, the queue index in handle_event should always be zero, but I think we could keep the checking on device_event here to ensure it's a validate event.